cli/Cargo.toml

```diff
@@ -64,7 +64,7 @@ once_cell = "1.12.0"
 deno_runtime = { version = "0.122.0" }
 deno_core = { version = "0.199.0" }
 deno_ast = { version = "0.27.2", features = ["transpiling"] }
-birdcage = { version = "0.3.1" }
+birdcage = { version = "0.3.1", path = "../../birdcage" }
```

Comment:
Is this path entry left over from local development?

Reply:
That's why this is still a draft, will need to wait for the birdcage PR to be merged.